sass/includes/_header.scss

```diff
@@ -269,6 +277,10 @@
         }
       }
     }
+
+    @media only screen and (min-width: $screen__md) {
+        display: none;
```

Comment:
I've just added this one small change to make sure the mobile menu input is hidden on desktop as well as the label.